Makefile

```diff
@@ -84,8 +84,8 @@ REQ_FILES = \
 	requirements/edx/coverage \
 	requirements/edx/doc \
 	requirements/edx/paver \
-	requirements/edx-sandbox/shared \
 	requirements/edx-sandbox/py35 \
+	requirements/edx-sandbox/py38 \
```

Comment:
I don't see `edx-sandbox/py38.txt`, did you run `make upgrade` after adding this line?

Reply:
I missed adding `py38.txt` before running `make upgrade` due to which this got skipped. Ran `make upgrade` again on the PR to resolve this issue.